data/maps/EverGrandeCity_ChampionsRoom/scripts.inc

```diff
@@ -1,181 +1,181 @@
 EverGrandeCity_ChampionsRoom_MapScripts:: @ 82289EF
 	map_script MAP_SCRIPT_ON_TRANSITION, EverGrandeCity_ChampionsRoom_OnTransition
-	map_script MAP_SCRIPT_ON_WARP_INTO_MAP_TABLE, EverGrandeCity_ChampionsRoom_MapScript2_228A05
-	map_script MAP_SCRIPT_ON_FRAME_TABLE, EverGrandeCity_ChampionsRoom_MapScript2_228A14
+	map_script MAP_SCRIPT_ON_WARP_INTO_MAP_TABLE, EverGrandeCity_ChampionsRoom_MapScript2_OnWarp
```

Comment:
You'll want to omit the `MapScript2` part of these labels, just like the previous PR of yours.  I think this is true for the other "document .incs" PR you have opened.